__tests__/example.test.ts

```diff
@@ -1,28 +1,116 @@
+              ],
+              "name": "Night light",
+              "nicknames": [
+                "wall plug"
+              ]
+            },
+            "willReportState": false,
+            "roomHint": "kitchen",
+            "deviceInfo": {
+              "manufacturer": "lights-out-inc",
+              "model": "hs1234",
+              "hwVersion": "3.2",
+              "swVersion": "11.4"
+            },
+            "otherDeviceIds": [
+              {
+                "deviceId": "local-device-id"
+              }
+            ],
+            "customData": {
+              "fooValue": 74,
+              "barValue": true,
+              "bazValue": "foo"
+            }
+          },
+          {
+            "id": "456",
+            "type": "action.devices.types.LIGHT",
+            "traits": [
+              "action.devices.traits.OnOff",
+              "action.devices.traits.Brightness",
+              "action.devices.traits.ColorSetting"
+            ],
+            "name": {
+              "defaultNames": [
+                "lights out inc. bulb A19 color hyperglow"
+              ],
+              "name": "lamp1",
+              "nicknames": [
+                "reading lamp"
+              ]
+            },
+            "willReportState": false,
+            "roomHint": "office",
+            "attributes": {
+              "colorModel": "rgb",
+              "colorTemperatureRange": {
+                "temperatureMinK": 2000,
+                "temperatureMaxK": 9000
+              },
+              "commandOnlyColorSetting": false
+            },
+            "deviceInfo": {
+              "manufacturer": "lights out inc.",
+              "model": "hg11",
+              "hwVersion": "1.2",
+              "swVersion": "5.4"
+            },
+            "customData": {
+              "fooValue": 12,
+              "barValue": false,
+              "bazValue": "bar"
+            }
+          }
+        ]
+      }
+    }
+});
+
+// Test passes, using the defined schema 
+test('Sync Response passes', async () => {
   const exampleSyncRes = require('./example.sync.response.json');
   const noErrors = testlib.validate(exampleSyncRes, 'sync');
   expect(noErrors).toBe(undefined);
 });
 
-// Test fails and shows errors
-test('Test 2: Sync Response fails', async () => {
-  const exampleSyncResFail = require('./example.sync.response.fail.json');
-  const syncErrors = testlib.validate(exampleSyncResFail, 'sync');
-  console.log(syncErrors);
-  expect(syncErrors && syncErrors.join('\n')).toBe(undefined);
-});
-
-test('Test 3: Sync Response using actions an intent handler', async () =>{
+//Test passes, using intent handler (this currently shows an error though)
+test('Sync Response using actions an intent handler', async () =>{
   const reqSync = testreq.generateSyncRequest();
-  const res = app.handler(reqSync);
+  const res = app.onSync(reqSync)
   const testlibValid = testlib.validate(res, 'sync');
+  console.log(testlibValid)
```

Comment:
I don't believe we need to log these to the console.

Reply:
There's a few errors that occur here, so just left this in for debugging purposes right now. Shows that there's an error when there shouldn't be

Reply:
An error in the JSON response?

Reply:
Yeah it shows that it needs params object since I'm taking from the actions on google library SmartHomeApp, need to make a fake app instead

Reply:
Okay, let's make sure to add that in